### ISSUE TITLE
discriminator loss goes to infinity

### ISSUE THREAD

**Issue by 065294847** [lucidrains/phenaki-pytorch#29]
Hi,

I'm trying to train the cvivit on a set of 10000 images. The vae-loss keeps going down, but the discriminator loss keeps rising infinity. It's easy to fool :)

Any idea what the problem is?

**Comment by lucidrains** [comment]
hmm maybe 10k images is not enough

are you doing any augmentations?

**Comment by 065294847** [comment]
No augmentations.

**Comment by miguelvalente** [comment]
Check the gradients and output values at every step of the way to see where it starts to go wrong.

**Comment by 065294847** [comment]
Thanks. It starts to jump dramatically already at step 500.

**Comment by lucidrains** [comment]
@065294847 what is your batch size and learning rate?

**Comment by 065294847** [comment]
Just the defaults, batch size 4, lr 3e-4

**Comment by miguelvalente** [comment]
Try it with a dataset that's known to work and converge. Just to rule it out.

**Comment by 065294847** [comment]
Thanks, I'll experiment a bit and report back

**Comment by 065294847** [comment]
Thanks, I'll experiment a bit and report back

**Comment by lucidrains** [comment]
@065294847 try lowering your learning rate, or increase your effective batch size (either increase `batch_size` or `grad_accum_every`)

**Comment by lucidrains** [comment]
@065294847 get more data too, 10k is nothing... try 100k or a million

**Comment by 065294847** [comment]
Yes of course, just wanted to do a quick test first :)
